crates/interpreter/src/exec.rs

```diff
@@ -1021,12 +1028,11 @@ impl<'m> Thread<'m> {
     }
 
     fn pop_values(&mut self, n: usize) -> Vec<Val> {
-        let mut values = Vec::new();
-        for _ in 0 .. n {
-            values.push(self.pop_value());
-        }
-        values.reverse();
-        values
+        pop_values(n, || self.pop_value())
```

Comment:
I'm not sure why I wrote this function like that, but it seems `Vec::split_off` would work (it probably didn't exist back then).